### DISCUSSION TITLE
Embed Kanban board inside other notes?

### DISCUSSION THREAD

**Discussion by nfelger** [mgmeyers/obsidian-kanban#455] [Ideas & Feature Requests]
Is it possible to embed a Kanban board in other notes?

I'm have notes I'm using to track progress on (small, short-lived) projects. Each of them contains a list of tasks I need to work on to complete the project. I'd LOVE for each of these project notes to show the tasks in a little Kanban board. Is that possible?

I could imagine either inline boards or - if that isn't possible - embedding a board from another note using `![[]]`…

**Comment by nfelger** [comment]
After stumbling on the [roadmap](https://github.com/mgmeyers/obsidian-kanban/projects/1), I saw https://github.com/mgmeyers/obsidian-kanban/issues/4 and https://github.com/mgmeyers/obsidian-kanban/issues/21.

So if anyone else is looking for what I'm looking for: it doesn't exist yet, but it is being considered (albeit with a very low priority at the moment).

**Comment by MugenLee** [comment]
I want it too
Kanban can embed notes, and daily is very helpful for many scenarios that require task management, such as project management

But the author seems to have not updated the mainline function for a long time

I don't know if there is any problem

Hope it doesn't give up halfway, hope this doesn't happen

**Comment by senged** [comment]
Not sure if it is the same thing, but I'm just looking to have custom markdown content that is *not deleted* by the plugin. For example, I want to treat this board tagged as a "#project" in my system, and I have other content that would go along with this project. As I'm trying this out I find it just deletes my custom content in some cases, and in others it converts a section into task items.   

I don't see any settings or documentation on this.

**Comment by AaronKnowls** [comment]
I'm also very interested in using Kanban boards inside notes. Very much!

**Reply by JuanCarlosDuran** [reply]
Meet too, It will have the Kanba board + the todo list in the journal daily (Ass developer you can follow the tickets)

**Comment by bfloydd** [comment]
I would find more usage with this plugin if the board could be embedded within notes

**Reply by stoweboyd** [reply]
Transcluding kanbans using Make.md plugin means you can embed kanban boards that resolve as kanbans.
![Screenshot 2024-06-08 at 9 00 43 AM](https://github.com/mgmeyers/obsidian-kanban/assets/3037422/601bf06c-fac4-421c-9d97-0b435a2e5b2e)

**Reply by OogieM** [reply]
Can you explain exactly how you do this? I've never used the make.md plug-in and I'm not even sure which section to look at to start to learn about this. 

2024-08-17 Update
I was finally able to get make.md installed, and got this to work once but the buggy nature of make.md has forced me back out of using it. Creating the linbjks works once, in edit mode, then they disappear in reading mode along with the transclusions. back to editing mode and they are also gone. Total mess. The real solution is for the kanban plug-in to handle it;

**Comment by AaronKnowls** [comment]
Has anyone come up with a way to circumvent this?
Maybe through a css snippet, or something else I can't think of?
This would be so so useful, but I'm failing to come up with a way to achieve it.

**Comment by porvah** [comment]
Is there a way to use tasks to convert a kanban column into a tasks list in my daily note
my dream scenario is for example I have a "today" column in my kanban board. when I open obsidian I want my daily note generated and its template to import all cards under today column as a task list in that note.

**Reply by supertom44** [reply]
I use dataview to do something similar.
I have a project idea's kanban board and on my daily note I have it set up to show my current project.

```
dataview
TABLE WITHOUT ID
	Tasks.text
FROM "Woodworking/Project Ideas.md"
FLATTEN file.tasks as Tasks
WHERE meta(Tasks.section).subpath = "Doing"
```

The kanban plugin creates heading for each column, so in my case it's looking for the heading Doing and showing all tasks under that heading.

**Reply by deadrubbish** [reply]
Hi @supertom44 - this looks great, an elegant solution, however I'm having a little trouble replicating. I may be over-complicating it. 
Are you able to give an overview of the structure of your 'Project Ideas' kanban ? 
Thanks
